pkg/backend/openshiftcluster/adminupdate.go

```diff
@@ -6,14 +6,15 @@ package openshiftcluster
 import (
 	"context"
 
+	"github.com/Azure/ARO-RP/pkg/api"
 	"github.com/Azure/ARO-RP/pkg/install"
 )
 
-func (m *Manager) AdminUpdate(ctx context.Context) error {
+func (m *Manager) AdminUpdate(ctx context.Context, registeredFeatures []api.RegisteredFeatureProfile) error {
 	// m.ocDynamicValidator.Dynamic is not called so that it doesn't block an
 	// admin update
 
-	i, err := install.NewInstaller(ctx, m.log, m.env, m.db, m.billing, m.doc)
+	i, err := install.NewInstaller(ctx, m.log, m.env, m.db, m.billing, m.doc, registeredFeatures)
```

Comment:
probably pass through m.subscriptionDoc and store that as a field of `type Installer struct`